src/foundry/client/pixi/groups/effects.d.mts

```diff
@@ -25,12 +20,12 @@ declare global {
     /**
      * A mapping of light sources which are active within the rendered Scene.
      */
-    lightSources: Collection<LightSource>;
+    lightSources: Collection<foundry.canvas.sources.PointLightSource>;
 
     /**
      * A Collection of vision sources which are currently active within the rendered Scene.
      */
-    visionSources: Collection<VisionSource>;
+    visionSources: Collection<foundry.canvas.sources.PointVisionSource<any>>;
```

Comment:
Can we avoid `any` here by using `PointVisionSource.VisionSourceData` instead?

I've considered writing something like `XYZ.Unknown` as a helper type analagous to `XYZ.Any` but safer like how `unknown` is safer than `any` but I can't figure out the right name and in some classes it's not sensible to use (because of conditional types making the types invariant etc.)